### PR TITLE
Add Droidcon basic splash screen

### PR DESCRIPTION
**Pull request by cbonoz**

### CODE REVIEW

File: Droidcon-Boston/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,16 +11,17 @@
...
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
-        <activity android:name=".views.MainActivity" />
+        <activity
+            android:name=".views.MainActivity"
+            android:screenOrientation="sensorPortrait" />
```
Comment (cbonoz) [inline Droidcon-Boston/app/src/main/AndroidManifest.xml]: we should probably remove this at some point (currently agenda UI doesn't handle portrait very well)

Comment (nisrulz) [reply Droidcon-Boston/app/src/main/AndroidManifest.xml]: Valid point. We should have a different layout for landscape version if we are going to allow that.

Comment (ragdroid) [reply Droidcon-Boston/app/src/main/AndroidManifest.xml]: Portrait only works!

**Comment by cbonoz** [comment]
Using separate image view overlay works better for different screen sizes.

**Pull request merged**